src/screens/NationCreateContainer/NationCreateScreen/index.js

```diff
@@ -441,12 +452,13 @@ class CreateNation extends NavigatorComponent {
   }
 }
 
-CreateNation.propTypes = {
-  onCancelNationCreation: PropTypes.func.isRequired,
-  onResetNationCreation: PropTypes.func.isRequired,
-  onSaveNationDraft: PropTypes.func.isRequired,
-  onDeleteNationDraft: PropTypes.func.isRequired,
-  onSubmitNation: PropTypes.func.isRequired,
+CreateNation.defaultProps = {
+  navigator: null,
```

Comment:
Please remove that, since navigator is not optional on Props.

Reply:
Done ;)